Automattic/wp-calypso
# Framework: CSS: Get rid of legacy CSS sections

This pull request gets rid of the last few CSS files inside `assets/stylesheets/sections`.

We need to start splitting our CSS into sections that match our JS sections, so this is just tidying up things to help us move forward with that.
  
#### Testing instructions
  
1. Run `git checkout try/rearrange-css` and start your server
2. Open the [`Home` page](http://calypso.localhost:3000/)
3. Check that the notifications panel is hidden or that the styles for the  element with id `#wpnc-panel` are loaded
4. Open the domain search page (http://calypso.localhost:3000/domains/add/[domain]
5. Check that the styles for the .domain-search__content element are loaded
6. Open the list of posts: http://calypso.localhost:3000/posts/my/[domain]
7. Trash a post
8. Check that the styles for the `.updated-confirmation` element are loaded.
  
  
#### Reviews
